### PR TITLE
fix(condo): DOMA-2606 fixed ticket editing when property deleted

### PR DESCRIPTION
**Pull request by Alllex202**
When ticket has deleted property, then should disable edit button and show hint

Снимок экрана 2023-02-15 в 12 38 46

### CODE REVIEW

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 1]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
-                                                                <UserNameField user={get(ticket, ['createdBy'])}>
-                                                                    {({ name, postfix }) => (
-                                                                        <Typography.Text style={TICKET_CREATE_INFO_TEXT_STYLE}>
-                                                                            {name}
-                                                                            {postfix && <Typography.Text type='secondary' ellipsis>&nbsp;{postfix}</Typography.Text>}
+                                <Row gutter={MEDIUM_VERTICAL_GUTTER}>
+                                    <Col span={24}>
+                                        <Row gutter={[0, 40]}>
+                                            <Col xl={13} md={11} xs={24}>
+                                                <Row gutter={[0, 20]}>
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:284]: constant

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 2]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
-                                                            <Typography.Text style={TICKET_CREATE_INFO_TEXT_STYLE}>
-                                                                <Typography.Text style={TICKET_CREATE_INFO_TEXT_STYLE} type='secondary'>{TicketCreationDate}, {TicketAuthorMessage} </Typography.Text>
-                                                                <UserNameField user={get(ticket, ['createdBy'])}>
-                                                                    {({ name, postfix }) => (
-                                                                        <Typography.Text style={TICKET_CREATE_INFO_TEXT_STYLE}>
-                                                                            {name}
-                                                                            {postfix && <Typography.Text type='secondary' ellipsis>&nbsp;{postfix}</Typography.Text>}
+                                <Row gutter={MEDIUM_VERTICAL_GUTTER}>
+                                    <Col span={24}>
+                                        <Row gutter={[0, 40]}>
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:282]: constant

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 3]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
-                                    </Row>
-                                </Col>
+                                                                    )
+                                                                }
+                                                            </Col>
+                                                        </Row>
+                                                    </Col>
+                                                </Row>
+                                            </Col>
+                                            <Col xl={11} md={13} xs={24}>
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:330]: constant

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 4]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
+                                                <Row justify={isSmall ? 'center' : 'end'} gutter={[0, 20]}>
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:331]: constant

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 5]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
+                                                                    data-cy='ticket__status-select'
+                                                                />
+                                                            </Col>
+                                                        </Row>
+                                                    </Col>
+                                                    {
+                                                        statusUpdatedAt && (
+                                                            <Col>
+                                                                <Typography.Paragraph style={TICKET_UPDATE_INFO_TEXT_STYLE}>
+                                                                    {ChangedMessage}: {dayjs(statusUpdatedAt).format('DD.MM.YY, HH:mm')}
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:349]: useMemo

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 6]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
+                                                                    {SourceMessage} — {get(ticket, ['source', 'name'], '').toLowerCase()}
+                                                                </Typography.Text>
+                                                            </Col>
+                                                            <Col span={24}>
+                                                                {
+                                                                    !isResidentTicket && !canReadByResident && (
+                                                                        <Typography.Text type='secondary' style={TICKET_CREATE_INFO_TEXT_STYLE}>
+                                                                            <FormattedMessage
+                                                                                id='pages.condo.ticket.title.CanReadByResident'
+                                                                                values={{
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:314]: useMemo

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 7]
```diff
@@ -272,105 +277,115 @@ export const TicketPageContent = ({ ticket, refetchTicket, loading, organization
...
-                                                                        <Typography.Text style={TICKET_CREATE_INFO_TEXT_STYLE}>
-                                                                            {name}
-                                                                            {postfix && <Typography.Text type='secondary' ellipsis>&nbsp;{postfix}</Typography.Text>}
+                                <Row gutter={MEDIUM_VERTICAL_GUTTER}>
+                                    <Col span={24}>
+                                        <Row gutter={[0, 40]}>
+                                            <Col xl={13} md={11} xs={24}>
+                                                <Row gutter={[0, 20]}>
+                                                    <Col span={24}>
+                                                        <Typography.Title style={{ margin: 0 }} level={1}>{TicketTitleMessage}</Typography.Title>
```
Comment (akarjakin) [inline apps/condo/pages/ticket/[id]/index.tsx:286]: constant

**Review by AleX83Xpert — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**